packages/manager/modules/billing-components/src/components/services-actions/services-actions.controller.js

```diff
@@ -66,6 +66,11 @@ export default class ServicesActionsCtrl {
           { serviceName: this.service.serviceId },
         );
         break;
+      case SERVICE_TYPE.ALL_DOM:
+        this.resiliateLink = this.service.canResiliateByEndRule()
+          ? resiliationByEndRuleLink
+          : `${this.autorenewLink}/delete-all-dom?serviceId=${this.service.serviceId}&serviceType=${this.service.serviceType}`;
```

Comment:
Can we get this url by using `coreURLBuilder` ?

Reply:
this.autorenewLink is built using coreURLBuilder, check line num 8.